database/migrations/default/1744826895218_ems_remove_match_status_constraint/up.sql

```diff
@@ -0,0 +1,47 @@
+alter table ems__incidents
+drop constraint matched_crash_pks_non_null,
+add column created_by text not null default 'system',
+add column updated_by text not null default 'system';
+
+comment on column ems__incidents.matched_crash_pks is 'The IDs of multiple crashes that were found to match this record. Set via trigger.''';
```

Comment:
Removes the constraint check that only allowed `matched_crash_pks` when the match status was `multiple_matches_by_automation`.

By removing this constraint, we can link an EMS record to a person record, change its status from `multiple_matches_by_automation` to `matched_by_manual_qa`, and leave the `matched_crash_pks` column untouched.

These kinds of details have me thinking about https://github.com/cityofaustin/atd-data-tech/issues/21714